project/src/main/webapp/style.css

```diff
@@ -253,7 +253,28 @@ h3 {
     padding: 2vw;
     transform: rotate(45deg);
   }
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
+>>>>>>> 01044406221a7358587c5ed7f485a127d5590154
+
+  #trend-frequency-data {
+    font-size: 6.5vw;
+    top: 120%;
+    left: 17%;
+    max-width: 75vw;
+  }
+
+  #trend-frequency-data div {
+    margin: 6vw 0;
+  }
+}
+<<<<<<< HEAD
+=======
 }  
+>>>>>>> 2bacd37db9ed029c5d8d7b27aa02bf883256bc36
```

Comment:
Here is another part where you might just want to remove this leftover stuff from the merge